velox/dwio/parquet/writer/Writer.cpp

```diff
@@ -217,7 +218,20 @@ dwio::common::StripeProgress getStripeProgress(
  *
  * This method assumes each input `ColumnarBatch` have same schema.
  */
-void Writer::write(const VectorPtr& data) {
+void Writer::write(const VectorPtr& dataToWrite) {
+  VectorPtr data = dataToWrite;
+  if (data && schema_) {
```

Comment:
Can data be null? What happens in that case?

Should we disallow null schema?

Reply:
@mbasmanova The data is expected to be non-null, while the schema may be null. In case the schema is null, it will utilize the type found in the data.